### PR TITLE
update website link in README

### PR DESCRIPTION
**Pull request by ANSHIKA1806**
@RushanKhan1 @AnanyaNegi @Pranav016  I have updated the new link of website in README and done a few modifications according to you. Please go through those and accept my pull request. Let me know if any other changes you expect.

### CODE REVIEW

**Comment by ANSHIKA1806** [comment]
@RushanKhan1 @Pranav016 Sir please go through my pull request. I've updated the new website link in the README file.

File: README.md [hunk 1]
```diff
@@ -9,17 +17,21 @@
  <img src="assets/blogsitegiff.gif" align = "center"  width=600px/>
  </p>
 
-## About-
+# About-
```
Comment (Pranav016) [inline README.md:20]: Keep heading size symmetric with the headings below it. Also with one `#` it becomes bigger than the main heading.

File: README.md [hunk 2]
```diff
@@ -9,17 +17,21 @@
...
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=ALPHAVIO/BlogSite)](https://dependabot.com)  
 
 - 'Blog' and 'blogging' are now loosely used for content creation and sharing on social media, especially when the content is long-form and one creates and shares content on regular basis.
-<br/>
 
-- This is a dynamically updating Blog posting website developed primarily using Node Js with EJS template engine and Mongoose as ODM(Object Data Modeling library).  [Visit BlogSite](https://blogsite321.herokuapp.com/)
+- This is a dynamically updating Blog posting website developed primarily using Node Js with EJS template engine and Mongoose as ODM(Object Data Modeling library). 
+
+- We made it really, really simple for people to make a blog and put whatever they want on it.<br>
```
Comment (Pranav016) [inline README.md:31]: Redundant

File: README.md [hunk 3]
```diff
@@ -9,17 +17,21 @@
...
+
+# [Visit BlogSite](https://alphavio-daily-journal.herokuapp.com/)
+
```
Comment (Pranav016) [inline README.md:35]: There is one more link in the README below that needs to be updated

**Review by Pranav016 — CHANGES_REQUESTED** [review]
Few changes

**Comment by ANSHIKA1806** [comment]
@Pranav016 
Sir, I've changed README file according to you. Please review it and let me know if any further changes are required or accept my pull request. 
Thank you Sir.

File: README.md [hunk 4]
```diff
@@ -26,9 +27,7 @@ There are many reasons to start a blog for personal use and only a handful of st
 
 - 'Blog' and 'blogging' are now loosely used for content creation and sharing on social media, especially when the content is long-form and one creates and shares content on regular basis.
 
-- This is a dynamically updating Blog posting website developed primarily using Node Js with EJS template engine and Mongoose as ODM(Object Data Modeling library).
```
Comment (Pranav016) [inline README.md]: This line was already present, didn't need to be removed..

File: README.md [hunk 5]
```diff
@@ -1,15 +1,16 @@
...
-There are many reasons to start a blog for personal use and only a handful of strong ones for business blogging. Blogging for business, projects, or anything else that might bring you money has a very straightforward purpose – to rank your website higher in Google SERPs, a.k.a. increase your visibility.
-
-<br><br><br>
 
 <div align="center">
   <img src="assets/New Banner.jpg" />
 </div>
+<br>
+There are many reasons to start a blog for personal use and only a handful of strong ones for business blogging. Blogging for business, projects, or anything else that might bring you money has a very straightforward purpose – to rank your website higher in Google SERPs, a.k.a. increase your visibility.
+
```
Comment (Pranav016) [inline README.md:10]: I feel we are misguiding out writers trying to show money incentives..

File: README.md [hunk 6]
```diff
@@ -1,15 +1,16 @@
...
+<br><br><br>
```
Comment (Pranav016) [inline README.md:11]: too many breaks

File: README.md [hunk 7]
```diff
@@ -26,9 +27,7 @@ There are many reasons to start a blog for personal use and only a handful of st
 
 - 'Blog' and 'blogging' are now loosely used for content creation and sharing on social media, especially when the content is long-form and one creates and shares content on regular basis.
 
-- This is a dynamically updating Blog posting website developed primarily using Node Js with EJS template engine and Mongoose as ODM(Object Data Modeling library). 
-
-- We made it really, really simple for people to make a blog and put whatever they want on it.<br>
+<br>
 
 # [Visit BlogSite](https://alphavio-daily-journal.herokuapp.com/)
```
Comment (Pranav016) [inline README.md:32]: reduce size, add more `#`

**Review by Pranav016 — CHANGES_REQUESTED** [review]
too many commits with same commit messages, I'd suggest squash commits or make a new PR, just fix the links for now with 1 commit. (we need to keep commit history clean)

**Comment by Pranav016** [comment]
closing PR for  now, reopen if required

**Comment by Pranav016** [comment]
> @Pranav016
> Sir, I've changed README file according to you. Please review it and let me know if any further changes are required or accept my pull request.
> Thank you Sir.

Yes , I got your point and reviewed it but some things are not satisfactory. So you can open a new PR with a couple small changes I suggested and I'll merge.

**Pull request closed without merging**